pom.xml

```diff
@@ -163,9 +163,19 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-site-plugin</artifactId>
+				<version>3.7.1</version>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-project-info-reports-plugin</artifactId>
+				<version>3.0.0</version>
+			</plugin>
 			<plugin>
 				<artifactId>maven-surefire-plugin</artifactId>
-				<version>3.0.0-M5</version>
+				<version>3.0.0-M3</version>
```

Comment:
wont this break the tests?

Reply:
Testing and running the application worked on my machine and the github action on this branch includes testing 
so it shouldn't.
But we can keep it on M5 since its the most recent version and I'll check the dependencies in my maven settings.

Reply:
I kept it at M5 since the missing dependency seemed to be a local issue and the build works with the new version just fine